openmetadata-service/src/main/java/org/openmetadata/service/resources/databases/TableResource.java

```diff
@@ -383,7 +383,7 @@ public Response delete(
           boolean hardDelete,
       @Parameter(description = "Id of the table", schema = @Schema(type = "UUID")) @PathParam("id") UUID id)
       throws IOException {
-    return delete(uriInfo, securityContext, id, false, hardDelete);
+    return delete(uriInfo, securityContext, id, true, hardDelete);
```

Comment:
Please don't change this in the default value of the API... We'll need to pass this as a flag in the `ometa_api` delete call

Reply:
Yes